tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -971,7 +971,7 @@ def _convert(cli_parser: argparse.ArgumentParser, framework, args):
                     raise e
 
                 remove_tmp_onnx_model(out_dir)
-                return ov_model
+                return ov_model, None
```

Comment:
the second returned value is `argv`. It shoud not be `None` in this case

Reply:
argv parsing becomes later, after this return, if it should not be, then I can move it

Reply:
argv parsing becomes later, after this return, if it should not be, then I can move it

Reply:
yes, correct. But this is the second issue and parsing or at least valid arguments should be returned.
`None` is not correct value in this case.

Reply:
looks like it contains some parameters which are not accepted by cli parser, attempt to parse it before model creation:
```
AttributeError: 'Namespace' object has no attribute 'example_input'
```
@popovaan probably I need your help to understand this functionality design and exdpected behaviour